### ISSUE TITLE
Set up Netlify for Pull Request previews?

### ISSUE THREAD

**Issue by XhmikosR** [nodejs/nodejs.org#2466]
Or something else that has the same effect.

Currently one has to pull down the changes locally to see them in action which is far from optimal.

**Comment by XhmikosR** [comment]
@Trott can we do this please? It will really help with PRs. I can help setting it up after you add it to the repo.

**Comment by Trott** [comment]
> @Trott can we do this please? It will really help with PRs. I can help setting it up after you add it to the repo.

You can either open a separate issue in the nodejs/admin repo, or I can transfer this issue there.

**Comment by XhmikosR** [comment]
@Trott yeah, please move it where appropriate. I thought it belonged to this repo.

To clarify, I don't mind whatever solution we go with, we just need this to make our lives easier :)
